scripts/widgets.js

```diff
@@ -0,0 +1,214 @@
+/*global $ proofIntData splitControl makeControlDiv */
```

Comment:
Can we get a less generic name for this file? @chrismiceli may be able to provide better guidance on the JS-way of doing things but a general catch-all file for all widgets doesn't seem like the best approach.

Reply:
Thanks. Perhaps split it up even more so there are files text-view.js and view-splitter.js. It it possible a use could arise for a view splitter without the text widget.

Reply:
I like that idea!